wavenet_ops.py

```diff
@@ -32,6 +34,29 @@ def causal_conv(value, filter_, dilation, name='causal_conv'):
             restored = tf.nn.conv1d(padded, filter_, stride=1, padding='SAME')
         # Remove excess elements at the end
         result = tf.slice(restored,
-                         [0, 0, 0],
-                         [-1, tf.shape(value)[1], -1])
+                          [0, 0, 0],
+                          [-1, tf.shape(value)[1], -1])
         return result
+
+
```

Comment:
Something is strange in Github.

I want to comment on `wavenet_ops.py` L42
```


def mu_law_encode(audio, quantization_channels):
```
should be 
```

def mu_law_encode(audio, quantization_channels):
```